docs/source/frequently_asked_questions.ipynb

```diff
@@ -1558,6 +1558,57 @@
     "There are now features where time unit is different from the default of seconds, such as `TIME_SINCE_LAST(sessions.session_start, unit=days)`, and `TIME_SINCE_FIRST(sessions.session_start, unit=years)`."
    ]
   },
+  {
+   "cell_type": "markdown",
```

Comment:
I think this example might fit better under DFS section since it is more about calling the `ft.dfs` function